spec/controllers/scim_rails/scim_users_request_spec.rb

```diff
@@ -8,7 +8,7 @@
   def post_request(content_type = "application/scim+json")
     # params need to be transformed into a string to test if they are being parsed by Rack
 
-    post "/scim_rails/scim/v2/Users",
+    post "/scim/v2/Users",
```

Comment:
This had to be updated because of the change to where the engine was mounted in the dummy app.